gym_sts/rl/mvp.py

```diff
@@ -37,11 +37,17 @@ def check_rllib_bug(space: spaces.Space):
     headless=ff.Boolean(True),
     animate=ff.Boolean(False),
     build_image=ff.Boolean(False),
+    reboot_frequency=ff.Integer(20, "Reboot game every n resets."),
```

Comment:
This default is a little low, 50-100 is probably fine.

Reply:
Done.